src/watchful/__about__.py

```diff
@@ -1,3 +1,3 @@
 """Package meta information for watchful."""
 
-__version__ = "3.7.0"
+__version__ = "3.8.0"
```

Comment:
Let's not bump versions or release anything here. We can pin the git rev id in our tests to use this updated version.

Reply:
that's a good idea. Next commit will revert this.